addons/website_quote/models/sale_quote.py

```diff
@@ -107,6 +113,16 @@ def _inject_quote_description(self, values):
             values['website_description'] = product.quote_description or product.website_description or ''
         return values
 
+    _sql_constraints = [
+        ('accountable_product_id_required',
+            "CHECK(display_type IS NOT NULL OR (product_id IS NOT NULL AND product_uom_id IS NOT NULL))",
+            "Missing required fields on accountable sale quote line."),
```

Comment:
"Missing required product and UoM on accountable ... " ?

Reply:
Changed